conda-recipes/llvmdev/meta.yaml

```diff
@@ -54,7 +54,7 @@ requirements:
   host:
     # needed for llc at runtime
     - zlib # [not win]
-    - xar # [osx]
+    - xar # [osx and x86_64]
```

Comment:
Does OSX-arm64 not need `xar`?

Reply:
The package doesn't exist in the package repository for this platform.  I'm not entirely certain what it is for, and the tests seem to pass without it.

Reply:
Seems like it was added a while back as a workaround: https://github.com/numba/llvmlite/pull/348 (specifically: https://github.com/numba/llvmlite/pull/348/commits/73cf559a7ab6617b5bcd255d095fc901d3197dc4) it may not be applicable any more.

Reply:
I guess the question is... does `llvm-objdump` work on M1, as from https://github.com/numba/llvmlite/commit/73cf559a7ab6617b5bcd255d095fc901d3197dc4 it seems like that was what triggered adding it. I suppose though that this is also somewhat moot if the package doesn't exist and further that this is a first build out on this architecture!